src/connect-zbt-2/getting-started-with-thread/forming-a-new-thread-network-with-zbt-2.md

```diff
@@ -90,4 +90,4 @@ This guide shows you how to use the {{ productName }} as a dedicated Thread adap
 - [Add your Thread-based devices to Home Assistant](https://www.home-assistant.io/integrations/thread/#adding-a-thread-based-device-to-home-assistant)
 - [About different Thread networks](https://www.home-assistant.io/integrations/thread/#about-different-thread-networks)
 - [About Thread border routers](https://www.home-assistant.io/integrations/thread/#about-thread-border-routers)
-- [Making Home Assistant your first network](https://www.home-assistant.io/integrations/thread/#to-make-home-assistant-your-first-thread-network)
\ No newline at end of file
+- [Making Home Assistant your first Thread network](https://www.home-assistant.io/integrations/thread/#to-make-home-assistant-your-first-thread-network)
```

Comment:
note: I think "To make Home Assistant your first Thread network" is an interesting choice for the title, but I can't think of a much better one now.

As that's the title in the HA docs at the moment, this is fine.